model/src/main/java/org/alfresco/transform/registry/TransformRegistryHelper.java

```diff
@@ -35,10 +34,13 @@
 import java.util.Set;
 import java.util.function.Consumer;
 
-import org.alfresco.transform.client.model.config.TransformOption;
-import org.alfresco.transform.client.model.config.TransformOptionGroup;
-import org.alfresco.transform.client.model.config.TransformOptionValue;
-import org.alfresco.transform.exceptions.TransformException;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.emptySet;
+import static java.util.Map.Entry;
+import static java.util.stream.Collectors.toMap;
+import static org.alfresco.transform.common.RequestParamMap.SOURCE_ENCODING;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
 
 class TransformRegistryHelper
```

Comment:
If this class should not be instantiated, please provide a private constructor to hide the implicit public one.

Reply:
done